src/applications/discharge-wizard/components/v2/ReviewPage.jsx

```diff
@@ -37,22 +48,31 @@ const ReviewPage = ({ formResponses, router, viewedIntroPage }) => {
       if (formResponses[shortName] === null) {
         return null;
       }
-
-      const reviewLabel = answerReviewLabel(shortName, formResponses);
+      const reviewLabel = REVIEW_LABEL_MAP[shortName];
+      const reviewAnswer = answerReviewLabel(shortName, formResponses);
 
       return (
         reviewLabel && (
           <li
             key={shortName}
-            className="vads-u-margin-bottom--0 vads-u-padding-y--3 vads-u-padding-x--1p5 answer-review"
+            className="vads-u-margin-bottom--0 vads-u-padding-y--3 vads-u-padding-x--1p5 answer-review-box"
           >
-            {reviewLabel}
-            <va-link
-              class="hydrated vads-u-padding-left--2"
-              onClick={() => onEditAnswerClick(ROUTES[shortName])}
-              name={shortName}
-              text="Edit"
-            />
+            <div className="answer-review-label">
+              <div
```

Comment:
We should be using semantic markup here, perhaps a `<p>` tag instead of a `<div>` to contain this text.